src/XMakeTasks/Al.cs

```diff
@@ -292,7 +292,7 @@ override protected string ToolName
         {
             get
             {
-                return NativeMethodsShared.IsWindows ? "AL.exe" : "al";
+                return "al.exe";
```

Comment:
No, we use `al.exe` and since mono's `Process.CreateProcess` can handle that, it works fine.